en/streaming-integrator/docs/connectors/downloading-and-Installing-Siddhi-Extensions.md

```diff
@@ -55,14 +54,63 @@ To uninstall a Siddhi extension, delete the relevant extension JAR in the `<SI_H
 
 To manage Siddhi extensions via the command line, see the following topics.
 
-### Finding Siddhi extensions to install
+### Identifying the Siddhi extensions to install/uninstall
 
-The following are some actions that you are required to perform in order to determine which Siddhi extensions you need to install.
+The following are some actions that you are required to perform in order to determine which Siddhi extensions you need to install. Navigate to the `<SI_HOME>/wso2/tools/extension-installer` in order to issue these commands.
 
-|**Required Action**                                                        |**CLI Command**        |
-|---------------------------------------------------------------------------|-----------------------|
-|
+- **Viewing the list of extensions that are currently installed**
 
+    You can view the complete list of Siddhi extensions that are currently installed in your Streaming Integrator setup. All the extensions listed are completely installed with the dependencies.
+    
+    To perform this action, issue the appropriate command out of the following based on your operating system:
+    
+    - **For Windows**     : `extension-installer.bat list`
+    - **For Linux/MacOS** : `./extension-installer.sh list`
+    
+- **Viewing the installation status of all the supported Siddhi extensions**
 
+    You can view the complete list of Siddhi extensions supported for WSO2 Streaming Integrator together with the current installation status for each extension.
```

Comment:
Shall we also say that, extension's name is also shown here. So that the user will know what is meant by '<EXTENSION_NAME>' in below explained commands.